packages/fxa-react/components/Illustration/index.tsx

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ import React from 'react';
+ import Hearts from '../../images/account-verified-hearts.svg';
+
+
+
```

Comment:
I think your autosave/autoformatter might be off / not looking at our Prettier config. Should help reduce some of the formatting /diff in other files too.

Also, we've been [inlining most of our SVGs](https://github.com/mozilla/fxa/tree/main/packages/fxa-settings#working-with-svgs).

We can probably move this SVG into `fxa-settings/src/images/`. _If_ we're going to use this image on more than one page, we could create a component for it like you did here, but we might not want to do that until we use it twice. If we keep it as a component for now, we might want to name it something other than `Illustration` to better describe the component, and then `className` should probably be set to `undefined` (or you can use the `classnames` package to help`) and `altText` and `testId` should have defaults.